### ISSUE TITLE
Error in if ((crosscorr$rel.phantom.coeff >= 0) & (crosscorr$rel.phantom.coeff <  :    missing value where TRUE/FALSE needed

### ISSUE THREAD

**Issue by jfertaj** [kundajelab/atac_dnase_pipelines#49]
Hi, 

I have been running a few ATAQ files through your pipeline and all of them have run with no errors but one sample with 2 technical replicates, here is the command i have used

bds_scr D24 atac.bds -species hg19 -nth 25 -no_par -out_dir D24_noENCODE -title D24_noENCODE -fastq1_1 /home/jfertaj/atac-seq/data/WTCHG_367049_13_1.fastq.gz -fastq1_2 /home/jfertaj/atac-seq/data/WTCHG_367049_13_2.fastq.gz -fastq2_1 /home/jfertaj/atac-seq/data/WTCHG_367049_14_1.fastq.gz -fastq2_2 /home/jfertaj/atac-seq/data/WTCHG_367049_14_2.fastq.gz

The program runs until the QC step when I got the following error:

Error in if ((crosscorr$rel.phantom.coeff >= 0) & (crosscorr$rel.phantom.coeff <  : 
  missing value where TRUE/FALSE needed

Fatal error: atac.bds, line 941, pos 4. Task/s failed.

Creating checkpoint file: Config or command line option disabled checkpoint file creation, nothing done.

Any ideas?

**Comment by leepc12** [comment]
Can you post a full error log here?
This error happened in the cross correlation QC step. While we are looking at it, you can disable it with `-no_xcor`.

**Comment by jfertaj** [comment]
Here is the full log, let me know if you need it by email or another method, thanks a lot

```
[DATE] : Sun Apr 30 10:06:21 BST 2017
[HOST] : dhcp210.well.ox.ac.uk
[SCR_NAME] : D24.BDS
[BDS_PARAM] :  atac.bds -species hg19 -nth 25 -no_par -no_idr -out_dir D24_noENCODE -title D24_noENCODE -fastq1_1 /home/jfertaj/atac-seq/data/WTCHG_367049_13_1.fastq.gz -fastq1_2 /home/jfertaj/atac-seq/data/WTCHG_367049_13_2.fastq.gz -fastq2_1 /home/jfertaj/atac-seq/data/WTCHG_367049_14_1.fastq.gz -fastq2_2 /home/jfertaj/atac-seq/data/WTCHG_367049_14_2.fastq.gz



== git info
Latest git commit		: 037cf436c4088d119a580e63c46eacde89d16433 (Wed Apr 26 20:08:12 2017)
	Reading parameters from section (default) in file(/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/default.env)...


== configuration file info
Hostname			: dhcp210.well.ox.ac.uk
Configuration file		: 
Environment file		: /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/default.env


== parallelization info
No parallel jobs		: true
Maximum # threads 		: 25


== cluster/system info
Walltime (general)		: 5h50m
Max. memory (general)		: 7G
Force to use a system		: local
Process priority (niceness)	: 0
Retiral for failed tasks	: 0
Submit tasks to a cluster queue	: 
Unlimited cluster mem./walltime	: false


Info: Environments module not found on your system (e.g. /etc/profile.d/modules.sh). Ignoring shell env. parameters like '-mod'. 


== shell environment info
Conda env. 			: bds_atac
Conda env. for python3		: bds_atac_py3
Conda bin. directory		: 

Shell cmd. for init.		: if [[ -f $(which conda) && $(conda env list | grep bds_atac | wc -l) != "0" ]]; then source activate bds_atac; sleep 5; fi;  export PATH=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/.:/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/modules:/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/utils:${PATH}:/bin:/usr/bin:/usr/local/bin:${HOME}/.bds; set -o pipefail; STARTTIME=$(date +%s)

Shell cmd. for init.(py3)	: if [[ -f $(which conda) && $(conda env list | grep bds_atac_py3 | wc -l) != "0" ]]; then source activate bds_atac_py3; sleep 5; fi;  export PATH=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/.:/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/modules:/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/utils:${PATH}:/bin:/usr/bin:/usr/local/bin:${HOME}/.bds; set -o pipefail; STARTTIME=$(date +%s)

Shell cmd. for fin.		: TASKTIME=$[$(date +%s)-${STARTTIME}]; echo "Task has finished (${TASKTIME} seconds)."


== output directory/title info
Output dir.			: /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE
Title (prefix)			: D24_noENCODE
	Reading parameters from section (default) in file(/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/default.env)...
	Reading parameters from section (hg19) in file(/home/jfertaj/atac-seq/data/bds_pipeline_genome_data/bds_atac_species.conf)...


== species settings
Species				: hg19
Species file			: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/bds_atac_species.conf

Species name (WashU browser)	: hg19
Ref. genome seq. fasta		: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/male.hg19.fa
Chr. sizes file			: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/hg19.chrom.sizes
Black list bed			: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/wgEncodeDacMapabilityConsensusExcludable.bed.gz


== ENCODE accession settings
ENCODE experiment accession	: 
ENCODE award RFA			: 
ENCODE assay category		: 
ENCODE assay title		: 
ENCODE award				: 
ENCODE lab				: 
ENCODE assembly genome		: 
ENCODE alias prefix		: KLAB_PIPELINE


== report settings
URL root for output directory	: 
Genome coord. for browser tracks	: 


== align multimapping settings
# alignments reported for multimapping	: 0


== align bowtie2 settings
Bowtie2 index			: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/bowtie2_index/male.hg19.fa
Walltime (bowtie2)		: 47h
Max. memory (bowtie2)		: 12G


== adapter trimmer settings
Maximum allowed error rate for cutadapt	: 0.10
Minimum trim. length for cutadapt -m	: 5
Walltime (adapter trimming)		: 23h
Max. memory (adapter trimming)		: 12G


== postalign bam settings
MAPQ reads rm thresh.		: 30
Rm. tag reads with str.		: 
No dupe removal in filtering raw bam	: false
Walltime (bam filter)		: 23h
Max. memory (bam filter)	: 12G
Dup marker				: picard
Use sambamba markdup (instead of picard)	: false


== postalign bed/tagalign settings
Max. memory for UNIX shuf			: 12G


== postalign cross-corr. analysis settings
Max. memory for UNIX shuf			: 12G
User-defined cross-corr. peak strandshift	: -1
Extra parameters for cross-corr. analysis	: 


== callpeak macs2 settings
Genome size (hs,mm)		: hs
Walltime (macs2)		: 23h
Max. memory (macs2)		: 15G


== callpeak naiver overlap settings
Bedtools intersect -nonamecheck	: false


== callpeak etc settings
# of top peaks to pick up in peak files	: 500000


== IDR settings
Append IDR threshold to IDR out_dir	: false


== ATAQC settings
TSS enrichment bed		: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/ataqc/hg19_gencode_tss_unique.bed.gz
DNase bed for ataqc		: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/ataqc/reg2map_honeybadger2_dnase_all_p10_ucsc.bed.gz
Promoter bed for ataqc		: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/ataqc/reg2map_honeybadger2_dnase_prom_p2.bed.gz
Enhancer bed for ataqc		: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/ataqc/reg2map_honeybadger2_dnase_enh_p2.bed.gz
Reg2map for ataqc			: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/ataqc/dnase_avgs_reg2map_p10_merged_named.pvals.gz
Reg2map_bed for ataqc		: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/ataqc/reg2map_honeybadger2_dnase_all_p10_ucsc.bed.gz
Roadmap metadata for ataqc	: /home/jfertaj/atac-seq/data/bds_pipeline_genome_data/hg19/ataqc/eid_to_mnemonic.txt
Max. memory for ATAQC			: 15G
Walltime for ATAQC			: 47h


== atac pipeline settings
Type of pipeline			: atac-seq
Fastqs are trimmed?			: false
Align only				: false
# reads to subsample replicates (0 if no subsampling)	: 0
# reads to subsample for cross-corr. analysis 	: 25000000
No pseudo replicates			: false
No IDR analysis on peaks		: true
No ATAQC (advanced QC report)		: false
No Cross-corr. analysis			: false
Use CSEM for alignment			: false
Smoothing window for MACS2		: 150
DNase Seq				: false
IDR threshold				: 0.1
Use old trim adapters			: false
Force to use ENCODE3 parameter set	: false
Force to use ENCODE parameter set	: false

== checking atac parameters ...

Checking parameters and data files for ATAQC. 


== checking adapters to be trimmed ...
Replicate 1 adapters : automatically detected
Replicate 2 adapters : automatically detected


== checking input files ...

Rep1 fastq (PE) :
	/home/jfertaj/atac-seq/data/WTCHG_367049_13_1.fastq.gz
	/home/jfertaj/atac-seq/data/WTCHG_367049_13_2.fastq.gz
Rep2 fastq (PE) :
	/home/jfertaj/atac-seq/data/WTCHG_367049_14_1.fastq.gz
	/home/jfertaj/atac-seq/data/WTCHG_367049_14_2.fastq.gz
Distributing 25 to ... 
{1=14, 2=11}

Detected adapter for rep1 (PE) : CTGTCTCTTATA, CTGTCTCTTATA
discarding /opt/miniconda3/bin from PATH
prepending /opt/miniconda3/envs/bds_atac/bin to PATH
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
################
ChIP data: /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign.gz 
Control data: NA 
strandshift(min): -500 
strandshift(step): 5 
strandshift(max) 1500 
user-defined peak shift NA 
exclusion(min): 10 
exclusion(max): NaN 
num parallel nodes: 14 
FDR threshold: 0.01 
NumPeaks Threshold: NA 
Output Directory: /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1 
narrowPeak output file name: NA 
regionPeak output file name: NA 
Rdata filename: NA 
plot pdf filename: /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.plot.pdf 
result filename: /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.qc 
Overwrite files?: TRUE

Decompressing ChIP file
Loading required package: caTools
Reading ChIP tagAlign/BAM file /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign.gz 
opened /tmp/user/1001/RtmpdankZ2/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign398a36f07e82
00:01:16.852	ExecutionerLocal 'Local[41]': Tasks [Local[41]]		Pending: 0	Running: 1	Done: 0	Failed: 0
		| PID   | Task state        | Task name | Dependencies | Task definition                                                                                                                                                                                                                                                                                              | 
		| ----- | ----------------- | --------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | 
		| 14662 | running (RUNNING) | xcor rep1 |              | if [[ $(which run_spp_nodups.R 2> /dev/null | wc -l || echo) == "1" ]]; then RUN_SPP=$(which run_spp_nodups.R);; else RUN_SPP=$(which run_spp.R);; fi; Rscript ${RUN_SPP} -rf; -c=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign. | 

done. read 19293694 fragments
ChIP data read length 75 
[1] TRUE
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
use 'source("https://bioconductor.org/biocLite.R")' or
  'source("http://bioconductor.org/biocLite.R")' to update
  'BiocInstaller' after library("utils")
Calculating peak characteristics
00:02:17.869	ExecutionerLocal 'Local[41]': Tasks [Local[41]]		Pending: 0	Running: 1	Done: 0	Failed: 0
		| PID   | Task state        | Task name | Dependencies | Task definition                                                                                                                                                                                                                                                                                              | 
		| ----- | ----------------- | --------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | 
		| 14662 | running (RUNNING) | xcor rep1 |              | if [[ $(which run_spp_nodups.R 2> /dev/null | wc -l || echo) == "1" ]]; then RUN_SPP=$(which run_spp_nodups.R);; else RUN_SPP=$(which run_spp.R);; fi; Rscript ${RUN_SPP} -rf; -c=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign. | 

00:03:18.885	ExecutionerLocal 'Local[41]': Tasks [Local[41]]		Pending: 0	Running: 1	Done: 0	Failed: 0
		| PID   | Task state        | Task name | Dependencies | Task definition                                                                                                                                                                                                                                                                                              | 
		| ----- | ----------------- | --------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | 
		| 14662 | running (RUNNING) | xcor rep1 |              | if [[ $(which run_spp_nodups.R 2> /dev/null | wc -l || echo) == "1" ]]; then RUN_SPP=$(which run_spp_nodups.R);; else RUN_SPP=$(which run_spp.R);; fi; Rscript ${RUN_SPP} -rf; -c=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign. | 

00:04:19.898	ExecutionerLocal 'Local[41]': Tasks [Local[41]]		Pending: 0	Running: 1	Done: 0	Failed: 0
		| PID   | Task state        | Task name | Dependencies | Task definition                                                                                                                                                                                                                                                                                              | 
		| ----- | ----------------- | --------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | 
		| 14662 | running (RUNNING) | xcor rep1 |              | if [[ $(which run_spp_nodups.R 2> /dev/null | wc -l || echo) == "1" ]]; then RUN_SPP=$(which run_spp_nodups.R);; else RUN_SPP=$(which run_spp.R);; fi; Rscript ${RUN_SPP} -rf; -c=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign. | 

Error in if ((crosscorr$rel.phantom.coeff >= 0) & (crosscorr$rel.phantom.coeff <  : 
  missing value where TRUE/FALSE needed
Execution halted
Minimum cross-correlation value 0.1155747 
Minimum cross-correlation shift 1500 
Top 3 cross-correlation values NA 
Top 3 estimates for fragment length NA 
Window half size NA 
Phantom peak location 65 
Phantom peak Correlation 0.4245396 
Normalized Strand cross-correlation coefficient (NSC) NA 
Relative Strand cross-correlation Coefficient (RSC) NA 
Task failed:
	Program & line     : 'modules/postalign_xcor.bds', line 121
	Task Name          : 'xcor rep1'
	Task ID            : 'atac.bds.20170430_100625_726/task.postalign_xcor.xcor_rep1.line_121.id_10'
	Task PID           : '14662'
	Task hint          : 'if [[ $(which run_spp_nodups.R 2> /dev/null | wc -l || echo) == "1" ]]; then RUN_SPP=$(which run_spp_nodups.R);; else RUN_SPP=$(which run_spp.R);; fi; Rscript ${RUN_SPP} -rf; -c=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign.'
	Task resources     : 'cpus: 14	mem: -1.0 B	wall-timeout: 8640000'
	State              : 'ERROR'
	Dependency state   : 'ERROR'
	Retries available  : '1'
	Input files        : '[/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign.gz]'
	Output files       : '[/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.qc, /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.plot.pdf]'
	Script file        : '/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/atac.bds.20170430_100625_726/task.postalign_xcor.xcor_rep1.line_121.id_10.sh'
	Exit status        : '1'
	Program            : 
		
		# SYS command. line 123
		
		 if [[ -f $(which conda) && $(conda env list | grep bds_atac | wc -l) != "0" ]]; then source activate bds_atac; sleep 5; fi;  export PATH=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/.:/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/modules:/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/utils:${PATH}:/bin:/usr/bin:/usr/local/bin:${HOME}/.bds; set -o pipefail; STARTTIME=$(date +%s)
		
		# SYS command. line 126
		
		 if [[ $(which run_spp_nodups.R 2> /dev/null | wc -l || echo) == "1" ]]; then RUN_SPP=$(which run_spp_nodups.R); \
				    else RUN_SPP=$(which run_spp.R); \
				    fi
		
		# SYS command. line 132
		
		 Rscript ${RUN_SPP} -rf \
					-c=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign.gz -p=14 \
					-filtchr=chrM -savp=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.plot.pdf -out=/home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.qc  
		
		# SYS command. line 135
		
		 sed -r 's/,[^\t]+//g' /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.qc > /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.qc.tmp
		
		# SYS command. line 136
		
		 mv /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.qc.tmp /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.qc
		
		# SYS command. line 138
		
		 TASKTIME=$[$(date +%s)-${STARTTIME}]; echo "Task has finished (${TASKTIME} seconds)."
	StdErr (100000000 lines)  :
		discarding /opt/miniconda3/bin from PATH
		prepending /opt/miniconda3/envs/bds_atac/bin to PATH
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		Loading required package: caTools
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		use 'source("https://bioconductor.org/biocLite.R")' or
		  'source("http://bioconductor.org/biocLite.R")' to update
		  'BiocInstaller' after library("utils")
		Error in if ((crosscorr$rel.phantom.coeff >= 0) & (crosscorr$rel.phantom.coeff <  : 
		  missing value where TRUE/FALSE needed
		Execution halted
	StdOut (100000000 lines)  :
		################
		ChIP data: /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign.gz 
		Control data: NA 
		strandshift(min): -500 
		strandshift(step): 5 
		strandshift(max) 1500 
		user-defined peak shift NA 
		exclusion(min): 10 
		exclusion(max): NaN 
		num parallel nodes: 14 
		FDR threshold: 0.01 
		NumPeaks Threshold: NA 
		Output Directory: /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1 
		narrowPeak output file name: NA 
		regionPeak output file name: NA 
		Rdata filename: NA 
		plot pdf filename: /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.plot.pdf 
		result filename: /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/qc/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.cc.qc 
		Overwrite files?: TRUE
		
		Decompressing ChIP file
		Reading ChIP tagAlign/BAM file /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign.gz 
		opened /tmp/user/1001/RtmpdankZ2/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign398a36f07e82
		done. read 19293694 fragments
		ChIP data read length 75 
		[1] TRUE
		Calculating peak characteristics
		Minimum cross-correlation value 0.1155747 
		Minimum cross-correlation shift 1500 
		Top 3 cross-correlation values NA 
		Top 3 estimates for fragment length NA 
		Window half size NA 
		Phantom peak location 65 
		Phantom peak Correlation 0.4245396 
		Normalized Strand cross-correlation coefficient (NSC) NA 
		Relative Strand cross-correlation Coefficient (RSC) NA 

Fatal error: atac.bds, line 941, pos 4. Task/s failed.

Creating checkpoint file: Config or command line option disabled checkpoint file creation, nothing done.
```

**Comment by leepc12** [comment]
It very hard to figure it out with a log only, can you email (leepc12 at gmail dot com) me your /home/jfertaj/atac-seq/tools/atac_dnase_pipelines/D24_noENCODE/align/rep1/WTCHG_367049_13_1.trim.PE2SE.nodup.25M.tagAlign.gz?

**Comment by leepc12** [comment]
fixed in the commit e929fc8. please get the latest pipeline and try again
